packages/kit/src/components/WebView/InpageProviderWebView.tsx

```
@@ -23,15 +23,30 @@ const InpageProviderWebView: FC<IInpageProviderWebViewProps> = forwardRef(
     });
 
-    useImperativeHandle(ref, (): IWebViewWrapperRef => {
+    useImperativeHandle(ref, () => {
       const wrapper = {
         innerRef: iframeWebviewRef.current,
         reload: () => iframeWebviewRef.current?.reload(),
         loadURL: (url: string) => {
           iframeWebviewRef.current?.loadURL(url);
         },
+        sendMessageViaInjectedScript: (message: any) => {
+          if (iframeRef.current && iframeRef.current.contentWindow) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)